src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1213,7 +1213,7 @@ class ContentExplorer extends Component<Props, State> {
         this.setState({ isLoading: true });
         this.api.getFolderAPI().create(
             id,
-            name,
+            name.trim(),
```

Comment:
Adding trim() function to remove the trailing whitespace when create button has been clicked. Similar to how EUA handles this situation. Ref: https://git.dev.box.net/Box/EndUserApp/blob/11365198c9e285203372204f25bb8d4e387caf01/src/components/modals/create-folder-modal/components/CreateFolderModal.js#L249

Reply:
I noticed there's an existing bug on line 1197, could we add a `trim()` there as well? it doesn't look like that `if` would ever enter because of the `if` right before it.

https://github.com/box/box-ui-elements/blob/6c68ae94753a136b4f00cdd9af116f9ea8b1fae2/src/elements/content-explorer/ContentExplorer.js#L1197

this way it'll be a little similar to rename functionality:
https://github.com/box/box-ui-elements/blob/6c68ae94753a136b4f00cdd9af116f9ea8b1fae2/src/elements/content-explorer/ContentExplorer.js#L1134

Reply:
resolved!